tests/integration/test_k8s.py

```diff
@@ -44,7 +45,19 @@ async def ready_nodes(k8s, expected_count):
 
 
 @pytest.mark.abort_on_fail
-async def test_nodes_ready(kubernetes_cluster):
+async def test_nodes_ready(kubernetes_cluster: model.Model):
     """Deploy the charm and wait for active/idle status."""
     k8s = kubernetes_cluster.applications["k8s"]
     await ready_nodes(k8s.units[0], 3)
+
+
+@pytest.mark.abort_on_fail
+async def test_remove_units(kubernetes_cluster: model.Model):
+    """Deploy the charm and wait for active/idle status."""
+    k8s: application.Application = kubernetes_cluster.applications["k8s"]
```

Comment:
This will spin up a whole new k8s cluster, right? 
What is the best practice here - should we merge both tests or is do we accept the additional testing time?

Reply:
nope, it should reuse the existing fixture since it has module scope, it will apply to all the tests in this python module